build.xml

```diff
@@ -40,40 +40,58 @@
   </target>
 
   <!-- Report rule violations with PHPMD (mess detector) -->
-  <target name="phpmd">
-    <exec command="${srcdir}/vendor/bin/phpmd ${srcdir}/src xml ${srcdir}/tests/phpmd.xml --exclude ${srcdir}/tests --reportfile ${builddir}/reports/phpmd.xml" />
+   <target name="phpmd">
+    <exec executable="${srcdir}/vendor/bin/phpmd">
+      <arg line="${srcdir}/src xml ${srcdir}/tests/phpmd.xml --exclude ${srcdir}/tests --reportfile ${builddir}/reports/phpmd.xml" />
+    </exec> 
   </target>
 
   <!-- PHP_Depend code analysis -->
-  <target name="pdepend">
-    <exec command="${srcdir}/vendor/bin/pdepend --jdepend-xml=${builddir}/reports/jdepend.xml --jdepend-chart=${builddir}/reports/dependencies.svg --overview-pyramid=${builddir}/reports/pdepend-pyramid.svg ${srcdir}/src" />
+   <target name="pdepend">
```

Comment:
This whole pdepend block is no longer used and should be removed. (Not related to your changes -- but we might as well clean it up while we're updating things).

Reply:
(The pdepend tool was discontinued last year... I removed it from composer.json and our Jenkins setup, but I must have forgotten to delete it from build.xml as well).